include/bouncer.h

```diff
@@ -498,9 +498,14 @@ struct PgUser {
 	uint8_t scram_ServerKey[32];
 	bool has_scram_keys;		/* true if the above two are valid */
 	bool mock_auth;			/* not a real user, only for mock auth */
+	bool dynamic_passwd;		/* does the password need to be refreshed every use */
 	int pool_mode;
 	int max_user_connections;	/* how much server connections are allowed */
 	int connection_count;	/* how much connections are used by user now */
+
+	/* cf_user points at the configured user that a user with a dynamic password is shadowing.
+	   For configured users, cf_user points at itself. */
```

Comment:
nit: for such multi line comments please use this style
```c
/*
 * some long comment
 * and some more lines
 */
```

(I'll try to figure out how to make uncrustify fix these automatically)